neonvm/tools/vm-builder/files/vminit

```diff
@@ -50,3 +50,6 @@ resize2fs /dev/vda
 # networking
 ip link set up dev lo
 ip link set up dev eth0
+
+# ssh
+/neonvm/bin/ssh-keygen -t ed25519 -f /etc/ssh/ssh_host_ed25519_key -N ""
```

Comment:
Do we need `-N ""` here? if so, why?

Reply:
We don't want to set a passphrase for our keys here. I'll add a comment.

Reply:
fixed in fe99970e9c88fe1c4fa450c5503324e726da41ac

Reply:
My understanding is that was already the case- what happens without it?

Reply:
VM gets blocked on this command

Reply:
Huh, funky. Fair enough